pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -212,10 +220,6 @@ void runCmd() throws Exception {
             }
             print("Updated successfully");
         }
-
```

Comment:
Why are we removing this?  The validation logic for update sink is different than that of create sink. By removing this logic, the validation would default to the one that is run for creating sinks which require more fields to be specified than an update.